lstchain/calib/camera/flatfield.py

```diff
@@ -43,6 +43,11 @@ class FlasherFlatFieldCalculator(FlatFieldCalculator):
         [-3, 3],
         help='Interval (number of std) of accepted charge standard deviation around camera median value'
     ).tag(config=True)
+    time_calibration_path = Unicode(
```

Comment:
Use `ctapipe.core.traits.Path` here, it also supports checking if the file exists.

Reply:
Ok